static/bidder-info/pgamssp.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://us-east.pgammedia.com/pserver"
+maintainer:
+  email: "info@pgammedia.com"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+    
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://cs.pgammedia.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
The redirect url is not working for me.

<img width="1722" alt="image" src="https://github.com/prebid/prebid-server/assets/87759626/78feeef8-a7cc-4c9c-9a10-9f139fb7adb4">

Reply:
Not working for me too.

Reply:
fixed, recheck please